src/app/selected-book/selected-book.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit, Input, OnChanges, SimpleChanges } from '@angular/core';
```

Comment:
Please move selected books to the Books Module, also move the associated route configuration